File: experience-trpg-backend/DTOs/MesaDto.cs
using System;
using System.Collections.Generic;

namespace experience_trpg_backend.DTOs
{
    public class MesaDto
    {
        public int MesaId { get; set; }
        public string? Nome { get; set; }
        public string? Descricao { get; set; }
        public DateTime DataCriacao { get; set; }
        public int CriadorId { get; set; }
        public int SistemaId { get; set; }
        public UsuarioDto? MesCriador { get; set; }
        public List<UsuarioDto>? Participantes { get; set; }
        public List<FichaDto> Fichas { get; set; } = new List<FichaDto>();
    }
}

File: TG-ExperienceTRPG-main/experience-trpg-frontend/src/app/dtos/mesa.dto.ts
export interface UsuarioDto {
    usuarioId: number; // ID do usuário
    nome: string;      // Nome do usuário
}

export interface MesaDto {
    mesaId: number;          // Identificador único da mesa
    nome: string;            // Nome da mesa
    descricao: string;       // Descrição da mesa
    dataCriacao: Date;      // Data em que a mesa foi criada
    criadorId: number;      // ID do criador da mesa
    sistemaId: number;      // ID do sistema associado à mesa
    convite: string;
    mesSistema?: string;     // Nome do sistema associado à mesa (opcional)
    mesCriador?: UsuarioDto; // Criador da mesa, opcional
    participantes?: UsuarioDto[]; // Lista de participantes, opcional
    imagem?: string;        // URL ou caminho da imagem da mesa (opcional)
    dataUltimoAcesso?: Date; // Data do último acesso (opcional)
}